README.md

```diff
@@ -2,18 +2,19 @@
 
 This package is a wrapper library for the [`@hashgraph/sdk`](https://www.npmjs.com/package/@hashgraph/sdk) library. It provides a means of integrating Hedera with WalletConnect with functions to facilitate implementing the [Hedera JSON-RPC schema](https://specs.walletconnect.com/2.0/blockchain-rpc/everscale-rpc). To see how to set up WalletConnect in dApps/wallets, view their [documentation](https://docs.walletconnect.com/2.0/). Once the dApp and wallet are configured to manage WalletConnect sessions, you can use this module's functions to easily create and handle sessions requests for Hedera.
 
+
 ## DApp Usage
 
-1. On the dApp side, we must first instantiate a WalletConnect [`SignClient`](https://docs.walletconnect.com/2.0/api/sign/dapp-usage) and establish a session with a wallet.
+1. **Initialization of WalletConnect SignClient**: Start by setting up a WalletConnect [`SignClient`](https://docs.walletconnect.com/2.0/api/sign/dapp-usage). This is your primary interface for establishing and managing sessions between your dApp and the user's wallet.
 
 ```js
 import SignClient from '@walletconnect/sign-client'
 
 const signClient = await SignClient.init({ ...signClientProps })
-// Perfrom other initialization steps. See WalletConnect docs.
+// Ensure other initialization steps are followed as per the WalletConnect documentation.
 ```
 
-2. Then we can build a transaction with the `@hashgraph/sdk`, making sure to set a `TransactionId` manually.
+2. **Constructing a Hedera Transaction**: Use the `@hashgraph/sdk` to build your desired Hedera transaction. It's important to manually set a `TransactionId` as it ensures traceability and potential reconciliation of transactions. [See the SDK documentation for more transaction types](https://docs.hedera.com/hedera/sdks-and-apis/sdks/transactions).
```

Comment:
I think setting a `TransactionId` is required when not using a signer (which we won't have)?